### ISSUE TITLE
validation error

### ISSUE THREAD

**Issue by a-jahani** [mrharicot/monodepth#56]
Training is not based on validation set and there is no validation error!!!

**Comment by mrharicot** [comment]
Indeed, but you can easily load the models saved at each epoch and test them on the validation splits.
You can also add a dataloader for validation data and add a validation loop at the end of each epoch.
